### PR TITLE
:sparkles: feat: detail tabs

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206703665848193

### CODE REVIEW

File: views/pool-details/index.tsx [hunk 1]
```diff
@@ -10,13 +10,22 @@ import { TOKEN_ICONS } from '@/lib';
 
 import PoolTitleBar from '../components/pool-title-bar';
 import Error from '../error';
-import { PoolDetailsProps } from './pool-details.types';
+import { PoolDetailsProps, PoolDetailsTabOption } from './pool-details.types';
+import PoolDetaislTabs from './pool-details-tabs';
```
Comment (KipandaJr) [inline views/pool-details/index.tsx:14]: has error in component name

File: views/pool-details/pool-details-tabs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <Box
+      height="3rem"
+      display="flex"
+      borderTopLeftRadius="inherit"
+      borderTopRightRadius="inherit"
+      borderBottom="0.01px solid #C6C6CA"
+    >
+      {items.map((el, index) => (
```
Comment (KipandaJr) [inline views/pool-details/pool-details-tabs/index.tsx:28]: change `el` to `item`

File: views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <Box
+      flex="1"
+      display="flex"
+      cursor="pointer"
+      onClick={onChange}
+      alignItems="center"
+      position="relative"
+      justifyContent="center"
+      borderBottom={isSelected ? '2px solid #0053DB' : ''}
```
Comment (KipandaJr) [inline views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx:20]: use the `ui-kit` color

File: views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import { FC } from 'react';
+
+import { PoolDetailsTabsItemProps } from './pool-details-tabs.types';
+
+const PoolsDetailsTabsItem: FC<PoolDetailsTabsItemProps> = ({
+  isSelected,
+  onChange,
+  item,
+}) => {
+  return (
```
Comment (KipandaJr) [inline views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx:11]: remove return

**Review by KipandaJr — CHANGES_REQUESTED** [review]
try use the same effect that we used on tabs, where the focus move slowly between inside tabs
image

File: views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx (hunk 3 above)
Comment (atilasousa) [reply views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx:20]: it dosen't exist in ui kit

Comment (KipandaJr) [reply views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx]: image
dont say that dude

**Review by KipandaJr — APPROVED** [review]

**Comment by git-marcopitra** [comment]
image
image
image

File: views/pool-details/pool-details-tabs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    onChangeTab?.(index);
+  };
+
+  return (
+    <Box
+      height="3rem"
+      display="flex"
+      borderTopLeftRadius="inherit"
+      borderTopRightRadius="inherit"
+      borderBottom="0.01px solid #C6C6CA"
```
Comment (git-marcopitra) [inline views/pool-details/pool-details-tabs/index.tsx:26]: wrong border

File: views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    flex="1"
+    display="flex"
+    cursor="pointer"
+    onClick={onChange}
+    alignItems="center"
+    position="relative"
+    justifyContent="center"
+    borderBottom={isSelected ? '2px solid #0053DB' : ''}
+  >
+    <Typography size="small" position="absolute" variant="body">
```
Comment (git-marcopitra) [inline views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx:21]: size `medium`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: views/pool-details/pool-details-tabs/index.tsx (hunk 5 above)
Comment (git-marcopitra) [reply views/pool-details/pool-details-tabs/index.tsx]: do not use static colors, try to use outline or outlineVariant

File: views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+}) => (
+  <Box
+    flex="1"
+    display="flex"
+    cursor="pointer"
+    onClick={onChange}
+    alignItems="center"
+    position="relative"
+    justifyContent="center"
+    borderBottom={isSelected ? '2px solid #0053DB' : ''}
```
Comment (git-marcopitra) [inline views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx:18]: Do not use static colors, instead use `primary`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+}) => (
+  <Box
+    flex="1"
+    display="flex"
+    cursor="pointer"
+    onClick={onChange}
+    alignItems="center"
+    justifyContent="center"
+    borderBottom={isSelected ? '2px solid' : ''}
+    borderBottomColor="primary"
```
Comment (git-marcopitra) [inline views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx:19]: isSelected ? 'primary' : 'transparent'

File: views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  item,
+}) => (
+  <Box
+    flex="1"
+    display="flex"
+    cursor="pointer"
+    onClick={onChange}
+    alignItems="center"
+    justifyContent="center"
+    borderBottom={isSelected ? '2px solid' : ''}
```
Comment (git-marcopitra) [inline views/pool-details/pool-details-tabs/pool-details-tabs-item.tsx:18]: remove the `isSelected` from here

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**